dbux-cli/src/util/buildBabelOptions.js

```diff
@@ -64,6 +70,9 @@ export default function buildBabelOptions(options) {
   //   injectDependencies();
   // }
 
+  if (!Array.isArray(packageWhitelist)) packageWhitelist = [packageWhitelist];
```

Comment:
Can we add a parser for comma-separated lists? E.g. `packageWhitelist.split(',').map(s => s.trim())`?

Reply:
Done.